src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,220 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //-------------------------------------------------
+  //Main Container
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
+    padding-top: $spacing-3xs; // allow space for focus styles
+  }
+
+  //-------------------------------------------------
+  //Main Container > Table Header
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    padding: rem(22px) 0 0 $spacing-md;
+  }
+
+  .#{$prefix}--data-table-v2-title {
+    @include type-style('productive-heading-03');
+    color: $text-01;
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    @include type-style('body-short-01');
+    color: $text-02;
+  }
+
+  //-------------------------------------------------
+  //Main Container > Table (Default Styles)
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+
+    thead {
+      @include type-style('body-short-01');
+      background-color: $ui-03;
+      font-weight: 700;
+    }
+
+    tbody {
+      @include type-style('body-short-01');
+      background-color: $ui-01;
+      width: 100%;
+    }
+
+    tr {
+      border: none;
+      height: rem(48px);
+      width: 100%;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) {
+      background: $hover-field;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) td {
+      color: $text-01;
+      background: $hover-field;
+      border-bottom: 1px solid $hover-field;
+      border-top: 1px solid $hover-field;
+    }
+
+    th,
+    td {
+      padding-left: $spacing-sm;
+      padding-right: $spacing-sm;
+      vertical-align: top;
+      text-align: left;
+      color: $text-02;
+      border-top: 1px solid $ui-01;
+      border-bottom: 1px solid $ui-01;
+    }
+
+    th {
+      color: $text-01;
+      border-top: 1px solid $ui-03;
+      border-bottom: 1px solid $ui-03;
+    }
+
+    th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+      padding-left: $spacing-lg;
+    }
+
+    th.#{$prefix}--table-column-icon:hover,
```

Comment:
What are these new classes for?

Reply:
The new classes were for new icon columns spec'd out for v10. They have specific padding that's different from other columns and the overflow menu column has a different padding when it's used alongside these other icons. Also the button size changes in different table-size variations, so I thought it would be best to add a class for these columns.
<img width="231" alt="Screen Shot 2019-03-20 at 10 32 53 AM" src="https://user-images.githubusercontent.com/32556167/54697649-ea520000-4afb-11e9-8a93-15af8e654d1d.png">

Reply:
> The new classes were for new icon columns spec'd out for v10.

Should we defer on these till post-v10 then since we are going for v9 parity first?